### PR TITLE
chore: remove encrypt=DANGER_PLAINTEXT

### PR DESCRIPTION
**Pull request by Jolg42**
Related https://github.com/prisma/docs/issues/2090

Only used in tests an not needed anymore.

### CODE REVIEW

File: .buildkite/publish/docker-compose.yml [hunk 1]
```diff
@@ -24,9 +24,9 @@ services:
       - TEST_MARIADB_URI=mysql://root:root@mariadb:3306/tests
       # SQL Server
       - TEST_MSSQL_URI=mssql://SA:Pr1sm4_Pr1sm4@mssql:1433/master
-      - TEST_MSSQL_JDBC_URI=sqlserver://mssql:1433;database=master;user=SA;password=Pr1sm4_Pr1sm4;trustServerCertificate=true;encrypt=DANGER_PLAINTEXT
-      - TEST_MSSQL_JDBC_URI_MIGRATE=sqlserver://mssql:1433;database=tests-migrate;user=SA;password=Pr1sm4_Pr1sm4;trustServerCertificate=true;encrypt=DANGER_PLAINTEXT
-      - TEST_MSSQL_SHADOWDB_JDBC_URI_MIGRATE=sqlserver://mssql:1433;database=tests-migrate-shadowdb;user=SA;password=Pr1sm4_Pr1sm4;trustServerCertificate=true;encrypt=DANGER_PLAINTEXT
+      - TEST_MSSQL_JDBC_URI=sqlserver://mssql:1433;database=master;user=SA;password=Pr1sm4_Pr1sm4;trustServerCertificate=true;
```
Comment (janpio) [inline .buildkite/publish/docker-compose.yml:27]: Should it maybe be this way instead?
```suggestion
      - TEST_MSSQL_JDBC_URI=sqlserver://mssql:1433;database=master;user=SA;password=Pr1sm4_Pr1sm4;trustServerCertificate=true
```

File: packages/migrate/src/__tests__/fixtures/schema-only-sqlserver/prisma/invalid-url.prisma [hunk 2]
```diff
@@ -1,4 +1,4 @@
 datasource my_db {
   provider = "sqlserver"
-  url      = "sqlserver://doesnotexist:1433;database=tests-migrate;user=SA;password=Pr1sm4_Pr1sm4;trustServerCertificate=true;encrypt=DANGER_PLAINTEXT"
+  url      = "sqlserver://doesnotexist:1433;database=tests-migrate;user=SA;password=Pr1sm4_Pr1sm4;trustServerCertificate=true;
```
Comment (janpio) [inline packages/migrate/src/__tests__/fixtures/schema-only-sqlserver/prisma/invalid-url.prisma:3]:
```suggestion
  url      = "sqlserver://doesnotexist:1433;database=tests-migrate;user=SA;password=Pr1sm4_Pr1sm4;trustServerCertificate=true;"
```

File: .buildkite/publish/docker-compose.yml (hunk 1 above)
Comment (janpio) [reply .buildkite/publish/docker-compose.yml:27]: cc @pimeys

File: packages/migrate/src/__tests__/fixtures/schema-only-sqlserver/prisma/invalid-url.prisma (hunk 2 above)
Comment (millsp) [reply packages/migrate/src/__tests__/fixtures/schema-only-sqlserver/prisma/invalid-url.prisma:3]: Should this have failed in the tests?

Comment (Jolg42) [reply packages/migrate/src/__tests__/fixtures/schema-only-sqlserver/prisma/invalid-url.prisma:3]: Looks like this one is not tested on SQL Server indeed.

File: .buildkite/publish/docker-compose.yml (hunk 1 above)
Comment (Jolg42) [reply .buildkite/publish/docker-compose.yml:27]: I don't think it matters?

**Review by millsp — APPROVED** [review]

**Pull request merged**